UI/window-basic-main.cpp

```diff
@@ -1570,6 +1576,15 @@ void OBSBasic::InitBasicConfigDefaults2()
 				  useNV ? SIMPLE_ENCODER_NVENC
 					: SIMPLE_ENCODER_X264);
 
+	const char *aac_default = "ffmpeg_aac";
+	if (EncoderAvailable("libfdk_aac"))
+		aac_default = "libfdk_aac";
+	else if (EncoderAvailable("CoreAudio_AAC"))
+		aac_default = "CoreAudio_AAC";
```

Comment:
Probably fine because libfdk is generally not available on platforms where we expect CoreAUdio, but this does change the priority from the current (CoreAudio > libfdk > MF > FFmpeg)

Reply:
I forgot to change that when I learn about it.